app/workers/rapid_ready_for_decision/form526_asthma_job.rb

```diff
@@ -34,5 +37,21 @@ def assess_data(form526_submission)
     def generate_pdf(_form526_submission, _assessed_data)
       raise 'method not implemented yet' # RuntimeError
     end
+
+    def meds_by_year_and_status_as_string(medication_requests)
+      medication_request_stats(medication_requests).reverse.map do |year, tallies|
+        "- #{year}: #{tallies}"
+      end.join("<br/>\n")
+    rescue
+      'Error formatting medications list!'
+    end
+
+    def medication_request_stats(medication_requests)
+      # Todo later: update HypertensionMedicationRequestData so that it can be used here
```

Comment:
`HypertensionMedicationRequestData` needs to be updated to not filter for only `active` medication requests.